src/java.base/share/classes/java/lang/String.java

```diff
@@ -4223,19 +4223,26 @@ private static int outdent(List<String> lines) {
      *     <td>continuation</td>
      *     <td>discard</td>
      *   </tr>
+     *   <tr>
+     *     <th scope="row">{@code \u005Cu...uXXXX}</th>
+     *     <td>unicode escape</td>
+     *     <td>single character UTF-16 equivalent</td>
```

Comment:
It would be clearer to have "single UTF-16 code unit equivalent" as the translation explanation

Reply:
Changed